src/port1.0/portutil.tcl

```diff
@@ -3372,8 +3372,13 @@ proc _check_xcode_version {} {
             }
 
             # Check whether /usr/include and /usr/bin/make exist and tell users to install the command line tools, if they don't
-            if {${os.major} <= 17 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
-                if {[vercmp $xcodeversion 10.0] >= 0} {
+            if {${os.major} <= 18 && (![file isdirectory [file join $cltpath usr include]] || ![file executable  [file join $cltpath usr bin make]])} {
+                if {${os.major} == 18} {
+                    ui_warn "System headers do not appear to be installed"
+                    ui_warn "You can install it as part of the Xcode Command Line Tools package by running `xcode-select --install'."
+                    ui_warn "Next install 'macOS_SDK_headers_for_macOS_10.14.pkg' that's included with the Command Line Tools package"
+                    ui_warn "'installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /'"
```

Comment:
I don't think it's useful to start complaining about this all the time. Only bring it up if a 32-bit build is actually requested and can't succeed.

Reply:
This one I’m really not sure on